Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2001?: Seems to come into existence about here.
+
+* 2009?: Some sort of overhaul occurs.
+
+***
+
+## Versions & Differences: 
+
+#### Tested:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+[^84759]: [Black Cats and Pointed Hats (USA) (En,De)](http://redump.org/disc/84759/)
+[^107929]: [Robinson Crusoe (USA)](http://redump.org/disc/107929/)
+[^81628]: [Webmaster (USA))](http://redump.org/disc/81628/)
+[^108150]: [Case for TKKG, A - The Mystery of the Mayan Treasure (USA) (En,De)](http://redump.org/disc/108150/)
+[^48393]: [Emergency 4 - Global Fighters for Life (Germany) (Disc 1)](http://redump.org/disc/48393/)
+[^82475]: [Sluzhba 911 (Russia)][Remnants](http://redump.org/disc/82475/)
+[^DE_EM2DX]: [German Emergency 2 Deluxe](waiting on null to upload/)
+[^Petterson]: [Petterson und Findus - Hühnertanz und Katerzirkus](waiting on null to upload)
+[^Kenny]: [Kenny's Adventure](waiting on null to upload)
+[^101786]: [Max and the Magician (Europe) (En,Fr,De,Es)](http://redump.org/disc/101786/)
+
+#### copy-X light:
+
+Basic ring protection via dummy files intersected by rings. No software protection is used unless otherwise supplied by the game itself. Can be circumvented by simply not copying the dummy files when copying the rest of the game from the disc.
+
+#### copy-X professional:
+
+Builds upon copy-X light. Contains everything described there, but also adds software protection in the form of a disc check, which appears to additionally check one of the dummy files. Copy-X professional also checks if the contemporaneously-popular kernel debugging software SoftICE is active via a process called SoftICE-Test, checking both for "SoftICE for Windows 95" and for "SoftICE for Windows NT", ending game execution if detected when booting the game.
+
+#### Untested:
+
+* copy-X audio[^D_8232013][^D_1436646][^D_2182905][^D_451419]
+* copy-X trial maker
+
+[^D_8232013]: [Die Wilden Kerle – DWK4 - Der Angriff Der Silberlichten (Das Hörspiel Zum Kinofilm)](https://www.discogs.com/release/8232013-Die-Wilden-Kerle-DWK4-Der-Angriff-Der-Silberlichten-Das-H%C3%B6rspiel-Zum-Kinofilm)
+[^D_1436646]: [Laava – Wherever You Are (I Feel Love)](https://www.discogs.com/release/1436646-Laava-Wherever-You-Are-I-Feel-Love)
+[^D_2182905]: [Manfred Mann* – 2006](https://www.discogs.com/release/2182905-Manfred-Mann-2006)
+[^D_451419]: [De-Phazz – Godsdog](https://www.discogs.com/release/451419-De-Phazz-Godsdog)
+
+#### copy-X audio:
+
+No samples have been tested at the moment, so exact specifics cannot be confirmed. Used to protect Audio CDs.
+
+#### copy-X trial maker:
+
+No samples currently known, and thus no information either.
+
+
+***
+
+## Samples:
+ 
+#### Confirmed:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+#### Unconfirmed: 
```

Comment:
Please add footnotes for why the developers/publishers mentioned are suspected to use copy-X. Also please rephrase these, as "many" is ambiguously defined and fairly relative. "Some" would be preferable, or something else if you'd prefer.

Reply:
Temporarily removed for the sake of getting this merged. The changes are still in my local repo,